### PR TITLE
Added test for professional education enrollment/payment

### PR DESCRIPTION
**Pull request by clintonb**
This test ensures payment is required for professional education courses.

XCOM-500

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @wedaly @jimabramson @rlucioni

File: acceptance_tests/test_profed_enrollment.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from acceptance_tests.pages import LMSCourseModePage
+
+
+@skip('Prof. Ed. tests should be run on an as-needed basis.')
+class ProfessionalEducationEnrollmentTests(EnrollmentApiMixin, LmsUserMixin, LoginMixin, WebAppTest):
+    def test_payment_required(self):
+        """ Verify payment is required before enrolling in a professional education course. """
+
+        # Note: Populate this list by querying the course modes/products for prof. ed. course IDs.
+        course_ids = ()
```
Comment (rlucioni) [inline acceptance_tests/test_profed_enrollment.py:15]: Just to be clear, is the expectation that this tuple will be updated manually when this test needs to be run?

Comment (clintonb) [inline acceptance_tests/test_profed_enrollment.py:15]: Correct.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**